src/NServiceBus.ContainerTests/When_using_nested_containers.cs

```diff
@@ -190,19 +203,52 @@ public void Should_not_dispose_singletons_when_container_goes_out_of_scope()
             }
         }
 
-        class SingletonComponent : ISingletonComponent, IDisposable
+        [Test]
+        public void Should_dispose_all_IDisposable_components_in_child_container()
```

Comment:
is that statement of the test name entirely correct? That's only true for container managed lifetimes, not for transient (instance per call) objects?